src/strings/unicode_ops.c

```diff
@@ -257,7 +257,7 @@ static void generate_property_codes_by_seq_names(MVMThreadContext *tc) {
         entry->name = (char *)uni_seq_pairs[num_names].name;
         entry->structindex = uni_seq_pairs[num_names].value;
         HASH_ADD_KEYPTR(hash_handle, property_codes_by_seq_names,
-            entry->name, strlen(entry->name), entry);
+            entry->name, strlen(entry->name), (MVMUnicodeNameRegistry *)entry);
```

Comment:
Wouldn't a more correct fix here be to change the type of `*property_codes_by_seq_names` above to `MVMUnicodeGraphemeNameRegistry *`?